author.hbs

```diff
@@ -1,33 +1,55 @@
 {{!< default}}
 {{! The tag above means - insert everything in this file into the {body} of the default.hbs template }}
 
+{{> "nav"}}
+
 {{! The big featured header }}
-<header class="main-header author-head {{#if author.cover}}" style="background-image: url({{author.cover}}){{else}}no-cover{{/if}}">
-    <nav class="main-nav overlay clearfix">
-        <a class="back-button icon-arrow-left" href="{{@blog.url}}">Home</a>
-        <a class="subscribe-button icon-feed" href="{{@blog.url}}/author/{{author.slug}}/rss/">{{author.name}}</a>
-    </nav>
+<header class="main-header no-cover">
 </header>
 
 <section class="author-profile inner">
+
     {{#if author.image}}
-    <figure class="author-image">
-        <div class="img" style="background-image: url({{author.image}})"><span class="hidden">{{author.name}}'s Picture</span></div>
-    </figure>
+
+    <div class="row">
+	    <figure class="small-8 columns author-image">
+	        <div class="img" style="background-image: url({{author.image}})">
+	            <span class="hidden">{{author.name}}'s Picture</span>
+	        </div>
+	    </figure>
+    </div>
+
     {{/if}}
+
     <h1 class="author-title">{{author.name}}</h1>
-    <h2 class="author-bio">{{author.bio}}</h2>
+
+	{{#if author.bio}}
+	    <div class="row">
+		    <p class="small-10 large-10 small-centered large-centered columns author-bio">
```

Comment:
Hmm. It's a tough call - here are some screenshots of various configurations:

Desktop Centered:
![center](https://cloud.githubusercontent.com/assets/5771615/6205670/9c8697b0-b52b-11e4-91fc-68bf3c05b005.png)


Desktop Left:
![left](https://cloud.githubusercontent.com/assets/5771615/6205673/a3b09ef0-b52b-11e4-8c23-f64cad90fa73.png)


Mobile Centered:
![center-mobile](https://cloud.githubusercontent.com/assets/5771615/6205675/a89a29fe-b52b-11e4-8435-3553de0215b1.png)

Mobile Left:
![left-mobile](https://cloud.githubusercontent.com/assets/5771615/6205680/d2feaddc-b52b-11e4-9dfd-bd13eef25a66.png)


I feel that given the minimalistic theme and the centered author metadata, it looks better centered. However, the final word is yours. :)